dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/DbFixture.cs

```diff
@@ -40,13 +40,22 @@ public async Task InitializeAsync()
     private IServiceProvider GetServices()
     {
         var services = new ServiceCollection();
-
+        var preventChangesToEntitiesInterceptor = new PreventChangesToEntitiesInterceptor<User, Guid>(TestUsers.All.Select(u => u.UserId), (user) => user.UserId);
```

Comment:
Can we use the Decorate approach like in `HostFixture` to eliminate this duplication?

Reply:
Done